backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/llvm/IrToBitcode.kt

```diff
@@ -684,7 +680,8 @@ internal class CodeGeneratorVisitor(val context: Context, val lifetimes: Map<IrE
             declaration?.scope() ?: llvmFunction!!.scope(0, subroutineType(context, codegen.llvmTargetData, listOf(context.irBuiltIns.intType)))
         }
 
-        override fun location(line: Int, column: Int) = scope?.let { LocationInfo(it, line, column) }
+        private val fileScope = (fileScope() as? FileScope)
+        override fun location(offset: Int) = scope?.let { scope -> fileScope?.let{LocationInfo(scope, it.file.fileEntry.line(offset), it.file.fileEntry.column(offset)) } }
```

Comment:
What exactly does this fix?

Reply:
I mean that probably we've got wrong file scope when calling `line`/`column` at call site. Could you point me at the specific code fragment that had wrong file scope?

Reply:
no we have it right for most of the block (it's returnable block), but when we calculate location for variable (captured by inliner) we could be in the middle of returnable block and file isn't actual for variable belonging to call site.    this exactly the place we used wrong place to calculate line/column position

Reply:
That's what I meant :)
Is it code context from `switchSymbolizationContextTo` that was affected by the fix?

Reply:
yes